CHANGELOG.md

```diff
@@ -24,7 +24,7 @@ or NaN
 - Updated package version information in ReadTheDocs documentation
 - Bumped dask from version 2024.5.2 to 2025.3.0 to fix a security issue (raised by @dependabot)
 - Bumped jinja2 from version 3.1.5 to 3.1.6 to fix a security issue (raised by @dependabot)
-- Edited installation instructions via Mamba/Conda to more accurately reflect the needed commands
+- Edited installation instructions via Mamba/Conda to more accurately reflect the needed commandsm
```

Comment:
Typo was added here (commands -> commandsm)

Reply:
Good catch!  Now fixed in https://github.com/geoschem/gcpy/pull/357/commits/80b8d5f225225110b387da3ed0a7817e6e0fe025